pandas/io/excel.py

```diff
@@ -448,72 +435,9 @@ def parse(self,
               convert_float=True,
               mangle_dupe_cols=True,
               **kwds):
```

Comment:
can we now renove the kwds?

Reply:
Will double check. There is one branch where these would get used and dispatched to the `TextParser`, though maybe that is dead code

Reply:
`**kwds` is passed through `TextParser` to the Python parser in `parsers.py`.

This is definitely not dead code, so I am very wary of removing this.  I think some more work can be done to better align the signature `read_excel` with that of `read_csv` (in the interest of creating a more unified data IO API)

IMO we should refrain from removing it (that would be an API IMO), especially as there is enough happening with the refactor.

Reply:
Yea after taking another look agreed with @gfyoung here. I think it's worth aligning the signatures of the different parse calls within the module and potentially removing keyword args if possible (I'm not actually sure what keywords would be applicable here) but would prefer to do in a separate PR since it would be potentially API breaking